addons/website_slides/static/src/js/slides_course_enroll_email.js

```diff
@@ -0,0 +1,82 @@
+odoo.define('website_slides.course.enroll.email', function (require) {
+'use strict';
+
+var core = require('web.core');
+var Dialog = require('web.Dialog');
+var publicWidget = require('web.public.widget');
+var _t = core._t;
+
+var SlideEnrollEmailDialog = Dialog.extend({
+    template: 'slide.course.email',
+
+    init: function (parent, options, modalOptions) {
+        modalOptions = _.defaults(modalOptions || {}, {
+            title: _t('Request course\'s access to responsible.'),
+            size: 'medium',
+            buttons: [{
+                text: _t('Yes'),
+                classes: 'btn-primary',
+                click: this._onSendEmail.bind(this)
+            }, {
+                text: _t('Cancel'),
+                close: true
+            }]
+        });
+        this.$element = options.$element;
+        this.channelId = options.channelId;
+        this._super(parent, modalOptions);
+    },
+
+    _onSendEmail: function () {
+        var self = this;
+        this._rpc({
```

Comment:
You can call directly methods in _rpc like
```
this._rpc({
    model: 'slide.channel',
    method: 'do_stuff',
    args:[ids, some parameter]
})
```

As controller does not add any value compared to calling the method itself, you can directly call the method. The other way would be to do some custom behavior in the controller like returning some error code but I don't think it has any real use here.